src/Host/Client/Impl/Definitions/IRSessionCallback.cs

```diff
@@ -35,7 +36,18 @@ public interface IRSessionCallback {
         /// Set locator mode in the plot window
         /// </summary>
         /// <returns>Location where the user clicked.</returns>
-        Task<LocatorResult> Locator(CancellationToken ct);
+        Task<LocatorResult> Locator(Guid deviceId, CancellationToken ct);
+
+        /// <summary>
+        /// Device is being created, so create/assign a plot window for it.
+        /// </summary>
+        /// <returns>Properties for the plot device window, such as width, height and resolution.</returns>
+        Task<PlotDeviceProperties> PlotDeviceCreate(Guid deviceId, CancellationToken ct);
```

Comment:
I'd rather add ```IPlotDevice: IDisposable``` and do ```Task<IPlotDevice> CreateDevice()``` and destroy is via `IPlotDevice.DestroyAsync()`. This will also eliminate need to expose guids.